config/notify.yml

```diff
@@ -0,0 +1,10 @@
+<% fake_api_key = "fake_api_key-00000000-0000-0000-0000-000000000000-00000000-0000-0000-0000-000000000000" %>
+
+development:
+  api_key: <%= ENV.fetch('GOVUK_NOTIFY_API_KEY', fake_api_key) %>
+
+test:
+  api_key: <%= ENV.fetch('GOVUK_NOTIFY_API_KEY', fake_api_key) %>
+
+production:
+  api_key: <%= ENV.fetch('GOVUK_NOTIFY_API_KEY', nil) %>
```

Comment:
I just had a thought - and I'm sorry it's inconsistent with our earlier conversation 🤕 - but I think it'd actually be consistent with our app if we moved this to our [secrets file](https://github.com/alphagov/content-publisher/blob/master/config/secrets.yml) where we already have a precedent.

Doing this would mean we could replace `YAML.safe_load(ERB.new(File.read(notify_config_path)).result).fetch(Rails.env)` with `Rails.application.secrets.notify_api_key`